cmake/onnxruntime_python.cmake

```diff
@@ -608,6 +608,16 @@ if (onnxruntime_USE_CUDA)
     )
 endif()
 
+if (onnxruntime_USE_ROCM)
```

Comment:
Does `onnxruntime_MINIMAL_BUILD` have any influence? Should ORTModule stuff be packed when minimal build is enabled?

Reply:
The changes here was copied form line #601 - #609. Since ROCm EP and CUDA EP have been made as so-called shared EP, which means the installation package is separated from other package. So, I guess that is the reason that onnxruntime_MINIMAL_BUILD wasn't checked in the section of #601 - #609 either.